README.md

```diff
@@ -32,7 +32,27 @@ usage: duct [-h] [--version] [-p OUTPUT_PREFIX]
             [-t {all,system-summary,processes-samples}]
             command [command_args ...] ...
 
-Gathers metrics on a command and all its child processes.
+duct is a lightweight wrapper that collects execution data for an arbitrary
+command.  Execution data includes execution time, system information, and
+resource usage statistics of the command and all its child processes. It is
+intended to simplify the problem of recording the resources necessary to
+execute a command to improve reproducibility, particularly in an HPC
+environment.
```

Comment:
```suggestion
execute a command, particularly in an HPC environment.
```

since AFAIK there is little to nothing about reproducibility aspect here.